src/enums/variant-payloads.md

```diff
@@ -9,8 +9,15 @@ You can define richer enums where the variants carry data. You can then use the
 
 <details>
 
+Key Points:
+  
+* This might be the first time seeing pattern matching so perhaps introduce the topic, more details are provided in further slides.
+  * Match is very similar to checking against an if expression. The difference is an if expression returns a boolean value, where this match expression can return any type.
+  * In a match, we check some expressions against some possible options, called match arms. Each option is declared with a pattern, an arrow, some code and then a comma. Sometimes a pair brackets between the arrow and the comma if it's multi-line.
```

Comment:
Do you mean "a pair of curly braces", i.e. a block?

Reply:
yes. :) I rerwote it